src/App.css

```diff
@@ -6,15 +6,23 @@
   align-items: center;
 }
 
+/* Nav header  */
 .logo-container {
   display: flex;
   align-items: center;
+  margin-top: 10px;
+  margin-left: 17px;
 }
```

Comment:
Please pull before pushing your branch 👍

Reply:
Hi @yaredtekle022 
This is an independent header component. I created the branch from dev following the correct gitflow. The only changes in this iteration are the horizontal line and logo and text alignment of the header.No other changes have been implemented.